### DISCUSSION TITLE
Welcome to dooing Discussions!

### DISCUSSION THREAD

**Discussion by atiladefreitas** [atiladefreitas/dooing#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by atiladefreitas** [comment]
Hi guys, you've been contributing a lot to the project recently, I'm loving it and I'm very grateful.

Thank you so much!

Today in particular, a lot of new features have been added to the project and I've just merged them.

Could you take a few minutes today and check if anything was lost or that could have caused a conflict? I'm asking for this because I just noticed that the confirmation window for deleting to-do items has disappeared in these updates.

I'll work on bringing it back later.

Thanks again for the contributions. This is my first time working on an open source project and I'm finding it really fun.

@clementpoiret @derivia

**Reply by clementpoiret** [reply]
Thanks for your message. I'll take some time to re-check my previous PRs to ensure I didn't accidentally delete prior changes from others :)
